pkg/controller/servicebindingrequest/sbrcontroller_test.go

```diff
@@ -94,25 +94,77 @@ func TestSBRControllerBuildGVKPredicate(t *testing.T) {
 
 	// update verifies whether only the accepted manifests trigger the reconciliation process
 	t.Run("update", func(t *testing.T) {
-		deploymentA := &appsv1.Deployment{}
```

Comment:
would the test with the old test fixture fail now? If not, let's keep it and introduce another test scenarios.

Reply:
The old tests check for ObjectMeta and according to https://github.com/redhat-developer/service-binding-operator/pull/597/files#diff-315611be703ec1f3ce8038660164858cL127, it ignores updates to CR status in which case metadata.Generation does not change,  hence watching for spec and status directly, here watching for meta changes is being ignored, as the meta changes were not capturing the status changes.
Hence for unit tests, ObjectMeta has been omitted from the structure.

Reply:
The above can be considered for altering and removing the ObjectMeta from following cases as well